test/controllers/update/update.managing.officer.controller.spec.ts

```diff
@@ -81,10 +87,12 @@ mockCompanyAuthenticationMiddleware.mockImplementation((req: Request, res: Respo
 const mockServiceAvailabilityMiddleware = serviceAvailabilityMiddleware as jest.Mock;
 mockServiceAvailabilityMiddleware.mockImplementation((req: Request, res: Response, next: NextFunction) => next());
 
+const mockLoggerDebugRequest = logger.debugRequest as jest.Mock;
 const mockSetApplicationData = setApplicationData as jest.Mock;
 const mockPrepareData = prepareData as jest.Mock;
 const mockMapFieldsToDataObject = mapFieldsToDataObject as jest.Mock;
 const mockSaveAndContinue = saveAndContinue as jest.Mock;
+const mockRemoveFromApplicationData = removeFromApplicationData as unknown as jest.Mock;
```

Comment:
do we need "as unknown"?

Reply:
tests seem to run fine without it, will take it out now!